src/common/dataHooks/useGamesSearch.js

```diff
@@ -75,6 +75,8 @@ export function useParachuteSearch({ perPage, kids }) {
 
   return {
```

Comment:
While you're in here, could you please change the return to include the whole useQuery object like our other dataHooks e.g.
`  return {
    ...response,
    puzzles: getPuzzles(response?.data),
  }`